driver/bpf/fillers.h

```diff
@@ -5776,6 +5784,108 @@ FILLER(sys_fchmod_x, true)
 	return bpf_val_to_ring(data, chmod_mode_to_scap(mode));
 }
 
+FILLER(sys_chown_x, true)
+{
+	/* Parameter 1: res (type: PT_ERRNO) */
+	long retval = bpf_syscall_get_retval(data->ctx);
+	int res = bpf_val_to_ring(data, retval);
+	CHECK_RES(res);
+
+	/* Parameter 2: path (type: PT_FSPATH) */
+	unsigned long path_pointer = bpf_syscall_get_argument(data, 0);
```

Comment:
Don't we need to resolve the user ptr and take the path here?

Reply:
And the others path pointers from `lchown`, `fchownat`

Reply:
Wow but they are tested :laughing: am i missing anything?

Reply:
we resolve them in the `bpf_val_to_ring` :)

Reply:
Whoa i completely forgot about that! Then it LGTM!